### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Collapse All_Keyboard: Keyboard focus is not staying on Collapse All button when Subscriptions are collapsed

### ISSUE THREAD

**Issue by Ashwinipola26** [microsoft/vscode-azurefunctions#2689]
### GitHubTags:
#Azure Tools for VS Code;#A11ySev2;#A11yTCS;#BM-Azure Tools for VS Code-MAC-Feb2021;#DesktopApp;#WCAG2.4.3;#MAC; #Keyboard;#AINotInScope

### OS Details:
macOS Catalina Version 10.15.7

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:45:54.515Z(6 days ago)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin X64 19.6.0
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab till Collapse All button and press enter
8. Check whether focus is staying on Collapse All button once the subscriptions are collapsed or else moving to next controls

### Actual Result:
Keyboard focus is not staying on Collapse All button when Subscriptions are collapsed, focus is moving to the next control.

### Expected Result:
Keyboard focus should be staying on Collapse All button when Subscriptions are collapsed

### User Impact:
Keyboard only users will have difficulty in tracking the exact focus once Subscriptions are being collapsed

### Attachment for Reference:

[Bug_2689_Keyboard focus is not staying on Collapse All button when Subscriptions are collapsed.zip](https://github.com/microsoft/vscode-azurefunctions/files/6002228/Bug_2689_Keyboard.focus.is.not.staying.on.Collapse.All.button.when.Subscriptions.are.collapsed.zip)

**Comment by ejizba** [comment]
This is not unique to our extension. Closing in favor of upstream VS Code issue: https://github.com/microsoft/vscode-azurefunctions/issues/2689
